onnx/backend/test/report/coverage.py

```diff
@@ -49,12 +49,30 @@ def add(self, node):  # type: (onnx.NodeProto) -> None
             self.attr_coverages[attr.name].add(attr)
 
 
+class ModelCoverage(object):
+    def __init__(self):  # type: () -> None
+        self.name = None  # type: Optional[Text]
+        self.node_coverages = defaultdict(NodeCoverage)  # type: Dict[Text, NodeCoverage]
```

Comment:
Wonder if you need to init self.graph as well? In case some people may use self.graph without initializing it.

Reply:
Absolutely!  I don't use it here, but I'll include it in case anyone does.  Thank you!